--- ./recorder/dogtail-recorder.desktop~	2006-08-22 02:03:30.000000000 +0800
+++ ./recorder/dogtail-recorder.desktop	2007-10-21 23:27:17.000000000 +0800
@@ -9,6 +9,6 @@
 Exec=dogtail-recorder
 Terminal=false
 Type=Application
-Icon=dogtail-head.svg
+Icon=dogtail-head
 StartupNotify=true
-Categories=Application;Development;
+Categories=Development;Building;Accessibility;Utility;
--- ./sniff/sniff.desktop~	2006-08-22 02:03:24.000000000 +0800
+++ ./sniff/sniff.desktop	2007-10-21 23:26:47.000000000 +0800
@@ -9,6 +9,6 @@
 Exec=sniff
 Terminal=false
 Type=Application
-Icon=dogtail-head.svg
+Icon=dogtail-head
 StartupNotify=true
-Categories=Application;Development;
+Categories=Development;Building;Accessibility;Utility;
